### PR TITLE
i made my project documentation

### PR DESCRIPTION
**Pull request by esxm**
### Pull Request Overview

This pull request adds the project documentation

### Testing Strategy

This pull request was tested by Alkhalidy Essam

### TODO or Help Wanted

This pull request still needs...

### Build

- [ ] Ran `npm build`.

### Author

Signed-off-by: Alkhalidy Essam <essam.al@stud.fils.upb.ro>

### CODE REVIEW

File: docs/project/2024/essam.al/index.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+|---------|-------------|-------|
+| [st7789](https://github.com/almindor/st7789) | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
+| [embedded-graphics](https://github.com/embedded-graphics/embedded-graphics) | 2D graphics library | Used for drawing to the display |
+
+## Links
+
+<!-- Add a few links that inspired you and that you think you will use for your project -->
+
+1. [link](https://example.com)
+2. [link](https://example3.com)
```
Comment (alexandruradovici) [inline docs/project/2024/essam.al/index.md:72]: Please add links for your project.

File: docs/project/2024/essam.al/index.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+|--------|--------|-------|
+| [Rapspberry Pi Pico W](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html) | The microcontroller | [35 RON](https://www.optimusdigital.ro/en/raspberry-pi-boards/12394-raspberry-pi-pico-w.html) |
+
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [st7789](https://github.com/almindor/st7789) | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
+| [embedded-graphics](https://github.com/embedded-graphics/embedded-graphics) | 2D graphics library | Used for drawing to the display |
```
Comment (alexandruradovici) [inline docs/project/2024/essam.al/index.md:65]: Do you need these libraries?

Comment (alexandruradovici) [inline docs/project/2024/essam.al/index.md:65]: Do you need these libraries?

File: docs/project/2024/essam.al/index.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
+# security door
```
Comment (alexandruradovici) [inline docs/project/2024/essam.al/index.md:1]:
```suggestion
# Security door
```

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Review by alexandruradovici — DISMISSED** [review]

**Review by alexandruradovici — APPROVED** [review]

**Review by DTeodor-Alexaandru — APPROVED** [review]

**Pull request merged**